apps/dashboard/src/components/workflow-editor/steps/email/maily.tsx

```diff
@@ -121,6 +121,9 @@ export const Maily = ({ value, onChange, className, ...rest }: MailyProps) => {
   const overrideTippyBoxStyles = () => (
     <style>
       {`
+          [data-tippy-root] {
+            z-index: 50 !important;
+          }
```

Comment:
fix the z-index issue with modals when they are shown over the maily "add" node button